modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -20,6 +20,14 @@ function inope_policy_review_menu() {
     'type' => MENU_LOCAL_TASK,
   );
 
+  $items['admin/reports/inope'] = array(
+    'title' => 'iNope Report',
+    'description' => 'Review of all policy nodes.',
+    'page callback' => 'inope_policy_review_report_page',
+    'access arguments' => array('administer inope policy review'),
+    'type' => MENU_LOCAL_TASK,
```

Comment:
is this just a regular menu item as well?  what makes it local tasky do you think?